web/client/components/TOC/__tests__/DefaultLayer-test.jsx

```diff
@@ -133,7 +133,7 @@ describe('test DefaultLayer module component', () => {
         const tool = ReactDOM.findDOMNode(TestUtils.scryRenderedDOMComponentsWithClass(comp, "glyphicon")[1]);
```

Comment:
You need to use a different index here to get the right tool, in this way you are simply skipping all the tests that were done before